pkg/ottl/grammar.go

```diff
@@ -211,9 +220,20 @@ func (i *editor) checkForCustomError() error {
 
 // converter represents a converter function call.
 type converter struct {
-	Function  string  `parser:"@(Uppercase(Uppercase | Lowercase)*)"`
-	Arguments []value `parser:"'(' ( @@ ( ',' @@ )* )? ')'"`
-	Keys      []Key   `parser:"( @@ )*"`
+	Function string `parser:"@(Uppercase(Uppercase | Lowercase)*)"`
+	// TODO Rename NewArguments to Arguments and refactor
+	Arguments    []value
+	NewArguments []argument `parser:"'(' ( @@ ( ',' @@ )* )? ')'"`
+	Keys         []Key      `parser:"( @@ )*"`
+}
+
+type argument struct {
+	Name  string `parser:"(@(Lowercase(Uppercase | Lowercase)*) Equal)?"`
```

Comment:
I just realized our tokens allow for function, and now parameter, names like `1` or `0`.  We should probably not allow that, but we can fix that in a separate PR.